c++/conductances/soplata/reticular/NaV.hpp

```diff
@@ -0,0 +1,68 @@
+// _  _ ____ _    ____ ___ _
+//  \/  |  | |    |  |  |  |
+// _/\_ |__| |___ |__|  |  |___
+//
+// fast sodium conductance for thalamic reticular cells
+// Soplata AE, McCarthy MM, Sherfey J, Lee S, Purdon PL, Brown EN, et al.
+// Thalamocortical control of propofol phase-amplitude coupling. PLOS
+// Computational Biology. 2017;13: e1005879. doi:10.1371/journal.pcbi.1005879
+
+#ifndef NAV
+#define NAV
+#include "conductance.hpp"
+
+//inherit conductance class spec
+class NaV: public conductance {
+
+public:
+
+    // specify parameters + initial conditions
+    NaV(double g_, double E_, double m_, double h_)
+    {
+        gbar = g_;
+        E = E_;
+        m = m_;
+        h = h_;
+
+        // defaults
+        if (isnan(gbar)) { gbar = 0; }
+        if (isnan (m)) { m = 0; }
+        if (isnan (h)) { h = 1; }
+        if (isnan (E)) { E = 30; }
+
+        p = 3;
+        q = 1;
+
+        // allow this channel to be approximated
+        approx_m = 1;
```

Comment:
same comment as before -- try including this in a model and set `approx_channels = 1`

and see if everything works; and also compare to when `approx_channels=0`

(if you use `closed_loop=false` you can overlay these voltage traces to make sure they're apprixmately the same)